cmd/kubeadm/app/constants/constants.go

```diff
@@ -96,6 +97,24 @@ const (
 
 	// MinExternalEtcdVersion indicates minimum external etcd version which kubeadm supports
 	MinExternalEtcdVersion = "3.0.14"
+
+	// DefaultEtcdVersion indicates the default etcd version that kubeadm uses
+	DefaultEtcdVersion = "3.0.17"
+
+	// MasterConfigurationConfigMap specifies in what ConfigMap in the kube-system namespace the `kubeadm init` configuration should be stored
+	MasterConfigurationConfigMap = "kubeadm-config"
```

Comment:
This merged in an other PR: https://github.com/kubernetes/kubernetes/pull/50320.
I kind of agree, but we can use different keys for different types of configuration, so I don't think it's a huge problem.
(Init config under the `MasterConfiguration` key, etc.)